telegram/ext/_application.py

```diff
@@ -624,11 +637,17 @@ def run_polling(
         On unix, the app will also shut down on receiving the signals specified by
         :paramref:`stop_signals`.
 
-        If :attr:`post_init` is set, it will be called between :meth:`initialize` and
-        :meth:`telegram.ext.Updater.start_polling`.
-
-        If :attr:`post_shutdown` is set, it will be called after both :meth:`shutdown`
-        and :meth:`telegram.ext.Updater.shutdown`.
+        The order of execution by `run_polling` is roughly as follows:
+        - :meth:`initialize`
+        - *:meth:`post_init`*
+        - :meth:`updater.start_polling`
+        - :meth:`start`
+        ...
```

Comment:
```suggestion
        - Run the application until the users stops it
```

Same below. I guess this is where the doc build fails :D

@harshil, why the `*` around the `post_*` methods? What is that supposed to render as?

Reply:
Wanted to emphasize it with a bold or italic, but I'm not sure if it will render it correctly

Reply:
ok let's leave it then